ci-operator/step-registry/sandboxed-containers-operator/env-cm/sandboxed-containers-operator-env-cm-ref.yaml

```diff
@@ -8,6 +8,9 @@ ref:
       cpu: 100m
       memory: 100Mi
   env:
+  - name: CATALOG_SOURCE_NAME
+    default: "redhat-operators"
+    documentation: "name of a catsrc that CI created for non-GA builds"
```

Comment:
Interesting, the documentation uses more cryptic words than the variable name :-) (I think it'd be nice to use full `The name of the catalog source...`)